pkgs/development/libraries/mbedtls/default.nix

```diff
@@ -11,14 +11,13 @@
 
 stdenv.mkDerivation rec {
   pname = "mbedtls";
-  name = "mbedtls-${version}";
-  version = "2.16.3"; # nixpkgs-update: no auto update
+  version = "2.16.9"; # nixpkgs-update: no auto update
```

Comment:
```suggestion
  version = "2.16.9";
```

Reply:
There was a reason to disable auto-updates: 0ee02a9d42b5fe1825b0f7cee7a9986bb4ba975d

Reply:
Oh that is a magic comment. Did not know that.